src/CsharpClient/QuixStreams.Streaming.UnitTests/States/StreamStateShould.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using FluentAssertions;
+using Microsoft.Extensions.Logging.Abstractions;
+using QuixStreams.State.Storage;
+using QuixStreams.Streaming.States;
+using Xunit;
+
+namespace QuixStreams.Streaming.UnitTests.States;
+
+public class StreamStateShould
+{
+    [Fact]
+    public void Constructor_ShouldCreateStreamState()
+    {
+        var streamState = new StreamState<int>(new InMemoryStorage(), key => -1, NullLoggerFactory.Instance);
```

Comment:
maybe we could rename `key => -1` to `defaultValue => -1`
it took me some time to figure out the test without the knowledge of StreamState implementation